plugins/imgur_a.js

```diff
@@ -76,7 +84,7 @@ hoverZoomPlugins.push({
                                                 caption += ';\n';
                                             }
                                             caption += img.description;
```

Comment:
I think that "null"s are appended on this line so it's better to check here rather than do a replace later.